libs/ui/components/inputs/search-icon/index.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useRef, useEffect } from 'react';
+import { Box, HStack, Input } from '@chakra-ui/react';
+import COLORS from '../../../constants/colors';
+import FONTS from '../../../constants/fonts';
+import Icon from '../../contents/icon';
+import SearchIconProps from './props';
+import ButtonIcon from '../button-icon';
+import { ButtonSize } from '../button-icon/props';
+import { motion } from 'framer-motion';
+export default function SearchIcon({
```

Comment:
```suggestion
const SearchInput = <T extends object>({...}) {...}

export default SearchInput
```
Where T is a generic type for forms